packages/Webkul/Shop/src/Resources/views/checkout/cart/coupon.blade.php

```diff
@@ -96,7 +96,7 @@
 
                 removeCoupon: function () {
                     let self = this;
-
+                    $('.cross-icon').css('display','none');
```

Comment:
No need for extra query selectors and all. Instead, disable the button till the response is successful.